Cubical/Categories/Displayed/Section/Base.agda

```diff
@@ -0,0 +1,306 @@
+{-
+   Local Sections of a displayed category.
+
+          A
+        ↗ |
+       /  |
+    M /   |
+     /    |
+    /     ↓
+   Δ ---→ Γ
+       γ
+
+   Every Section can be implemented as a Functorᴰ out of Unitᴰ (see
+   Displayed.Instances.Terminal):
+
+       M
+   Δ ----→ A
+   |       |
+   |       |
+   |       |
+   |       |
+   |       ↓
+   Δ ----→ Γ
+       γ
+
+   And vice-versa any Functorᴰ
+
+       M
+   B ----→ A
+   |       |
+   |       |
+   |       |
+   |       |
+   ↓       ↓
+   Δ ----→ Γ
+       γ
+
+   Can be implemented as a Section (see
+   Displayed.Constructions.TotalCategory)
+
+            A
+          ↗ |
+         /  |
+      M /   |
```

Comment:
Killer diagrams in this file btw

Reply:
Sanity checking myself: the variables in the diagrams and the variables in the definitions aren't the same, right?

Reply:
(The diagram variable names are presumably contexts/dependent types/context extensions?)

Reply:
(While the definitions are in terms of standard non-logic variable names.)
This unfortunately turned into three comments.

Reply:
you can edit comments lol

Reply:
And yes I used context and context extension syntax here. I guess I shouldn't